File: rudder-sdk-core/src/main/java/com/rudderstack/android/sdk/core/RudderLibraryInfo.java
package com.rudderstack.android.sdk.core;

import com.google.gson.annotations.SerializedName;
import com.rudderstack.android.sdk.core.BuildConfig;

class RudderLibraryInfo {
    @SerializedName("name")
    private String name = BuildConfig.LIBRARY_PACKAGE_NAME;
    @SerializedName("version")
    private String version = BuildConfig.VERSION_NAME;
}

File: models/src/main/java/com/rudderstack/models/android/RudderLibraryInfo.kt
package com.rudderstack.models.android

import com.google.gson.annotations.SerializedName

class RudderLibraryInfo(
    // = BuildConfig.LIBRARY_PACKAGE_NAME
    @SerializedName("name")
    private val name: String? = null,

    @SerializedName("version")
    private val version: String = "1.2.1",
)
